nss-ivory-trolls/WorkforceManagement
# Ac employee edit

git fetch --all
git checkout computers-filter
Run the application
Click the Computers item in the navigation
Search for a specific Make or Manufacturer and click submit
Confirm that the list of computers is reduced to only computers with the search from Make or Manufacturer

# Description
Employee Edit ticket complete. Added simple list of links to Home Page of application.

## Acceptance Criteria

**Given** user is viewing an employee
**When** user clicks on the Edit link
**Then** user should be able to edit the last name of the employee
**Or** change the department to which the employee is assigned
**Or** change the computer assigned to the employee
**Or** add/remove training programs for the employee to attend in the future

## Type of change
- [x ] New feature (non-breaking change which adds functionality)

# Testing Instructions for Change Made
** Run the following in SQL Database:
```
SET IDENTITY_INSERT Computer ON
INSERT INTO Computer (Id, PurchaseDate, DecomissionDate, Make, Manufacturer)
VALUES(0,GETDATE(), null, 'NO COMPUTER ASSIGNED', 'NONE')
SET IDENTITY_INSERT Computer OFF
```
git fetch --all
git checkout computers-filter
Run the application
Click the Employees item in the navigation
Edit an Employee
Ensure that the above Acceptance Criteria is met 

# Checklist:
